config/config.go

```diff
@@ -236,6 +236,29 @@ func (c *Config) UnmarshalYAML(unmarshal func(interface{}) error) error {
 				ogc.APIURL += "/"
 			}
 		}
+		for _, wcc := range rcv.WechatConfigs {
+			wcc.ApiURL = c.Global.WeChatAPIURL
+			if wcc.ApiURL == "" {
+				if c.Global.WeChatAPIURL == "" {
+					return fmt.Errorf("no global Wechat URL set")
+				}				
+			}
+			wcc.ApiSecret = c.Global.WeChatAPISecret
```

Comment:
Got be consistent, using `APISecret` on the global config but `ApiSecret` on the local one, better be `APISecret`